lib/scuba/wrapper.js

```diff
@@ -36,6 +40,9 @@ class ScubaClientImpl extends ScubaClient {
                     description: err.message,
                 });
             }
+            monitoring.utilizationServiceHealth.labels({
+                state: 'unhealthy',
+            }).inc();
```

Comment:
Incrementing counter is not ideal here if you want to have an accurate healthy percentage in your panel.
If scuba goes up and down multiple times, you'll have counters that keep incrementing.

You might want to use a gauge to set a boolean like this:
```js
// success
utilizationServiceHealth.labels(healthy).set(1)
utilizationServiceHealth.labels(unhealthy).set(0)
// error
utilizationServiceHealth.labels(healthy).set(0)
utilizationServiceHealth.labels(unhealthy).set(1)
```

Reply:
Done in https://github.com/scality/cloudserver/pull/5556/commits/8ef63ab5cd687bbe34c0a5eb42f8d56821a641e2